parm/soca/letkf/letkf_stage.yaml.j2

```diff
@@ -0,0 +1,74 @@
+######################################
+# set some variables
+######################################
+{% set gPDY = previous_cycle | to_YMD %}
+{% set gcyc = previous_cycle | strftime("%H") %}
+{% set PDY = current_cycle | to_YMD %}
+{% set cyc = current_cycle | strftime("%H") %}
+######################################
+# create working directories
+######################################
+mkdir:
+- "{{ DATA }}/bkg"
+- "{{ DATA }}/data_output"
```

Comment:
to be consistent with the deterministic, data_output should be Data

Reply:
changed